frigate/video.py

```diff
@@ -38,6 +38,9 @@
 
 def filtered(obj, objects_to_track, object_filters):
```

Comment:
You should include the ratio object filter here. This prevents them from being tracked.